yarn.lock

```diff
@@ -5033,9 +5033,9 @@ can-use-dom@^0.1.0:
   integrity sha1-IsxKNKCrxDlQ9CxkEQJKP2NmtFo=
 
 caniuse-lite@^1.0.30000981, caniuse-lite@^1.0.30000989, caniuse-lite@^1.0.30001017, caniuse-lite@^1.0.30001043, caniuse-lite@^1.0.30001061:
-  version "1.0.30001066"
-  resolved "https://registry.yarnpkg.com/caniuse-lite/-/caniuse-lite-1.0.30001066.tgz#0a8a58a10108f2b9bf38e7b65c237b12fd9c5f04"
-  integrity sha512-Gfj/WAastBtfxLws0RCh2sDbTK/8rJuSeZMecrSkNGYxPcv7EzblmDGfWQCFEQcSqYE2BRgQiJh8HOD07N5hIw==
+  version "1.0.30001157"
+  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001157.tgz"
+  integrity sha512-gOerH9Wz2IRZ2ZPdMfBvyOi3cjaz4O4dgNwPGzx8EhqAs4+2IL/O+fJsbt+znSigujoZG8bVcIAUM/I/E5K3MA==
```

Comment:
I wonder what this change is. 🤔

Reply:
oh poopers. let me see

Reply:
Oh I think it's bc I ran npx browserslist@latest --update-db

https://github.com/browserslist/browserslist/issues/492https://github.com/browserslist/browserslist/issues/492

Reply:
Is it still ok to merge, or should I edit?

Reply:
Ah, might bee fine then. 😅 I'd like to ask @sausman to weight in as well when he has a chance!

Reply:
@ShreyaThapa +1 should be fine